compiler/testSuite/Voptorder01.bal

```diff
@@ -0,0 +1,14 @@
+import ballerina/io;
+
+public function main() {
+    int? a = 5;
+    io:println(a <= (<int?>())); // @output false
```

Comment:
The cast won’t do what you think it does. You need another variable of type int?

Reply:
If I use a variable of type int? I get my operand as a `Register` where as if I do it like this I get it as `()`. I am using this test case to verify our approach works in either case.

Reply:
What's the point of the cast? You'll get it as `()` if you write `a <= ()`.